src/lti/LtiProvider.tsx

```diff
@@ -47,12 +46,6 @@ const LtiProvider = ({ locale: propsLocale }: Props) => {
   });
   const { t, i18n } = useTranslation();
   const locale = propsLocale ?? i18n.language;
-  const subjects = searchSubjects(searchParams.query);
```

Comment:
Er det meningen at subjects skal vises som søkeresultat for lti? det har det ikke gjort tidligere, men usikker på om det er bug eller ønsket oppførsel 😅

Reply:
Hvis jeg skulle gjettet skal vi nok ikke vise fag i LTI i det hele tatt. Formålet med LTI er jo å sette inn ressurser, så det gir vel ikke verdi å vise frem hverken emner eller fag. @gunnarvelle

Reply:
Emner har vi alltid vist, for det er jo artikler. Men fag gir ikkje meining å kunne sette inn.